docker/images/master/k8s/launch_test.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
```

Comment:
To keep the distinction between test and actual code clear, we should consider moving this file and teardown.sh to another folder, perhaps `/test/`